tests/test-helphub-read-time.php

```diff
@@ -10,11 +10,45 @@
  */
 class Helphub_Read_Time_Test extends WP_UnitTestCase {
 
+
+	/**
+	 * Setup our test server, endpoints, and user info.
+	 */
+	public function setUp() {
+		parent::setUp();
+		$this->user = $this->factory->user->create( array(
```

Comment:
Can you move this line into `test_hh_calculate_and_update_post_read_time()`?
Because it is used only once for now.